### DISCUSSION TITLE
Add a flag to "payable" or "premium need" articles

### DISCUSSION THREAD

**Discussion by alancpazetto** [dailydotdev/daily#1359] [Feature Requests]
It would be very nice to have a flag in articles that need to be a premium user (like Medium) to read.

It's very frustrating to find a nice article title, click to read and cannot read it all because only first title is available, to read all we need to pay 150USD.

Even if possible, filter to don't show these kind of articles.

Maybe (just a thought about) punish this kind of article, thinking about to incentivate free knowledge share, and don't make Daily Dev a advertising hub.

Are there some direction for that? What community think about it?

**Comment by sabdailydev** [comment]
Hi @alancpazetto , thanks for raising this. 
All posts on daily.dev are free to read. Sometimes we don't have a way of filtering out the premium ones. Please feel free to report these and we will take them down. Thanks for understanding.

**Reply by alancpazetto** [reply]
No problem, I understand.

In fact, not at all posts are free to read.

For example, [this post](https://app.daily.dev/posts/10-advanced-open-source-command-line-tools-enhance-your-productivity-hrkqhil1m) link to [this article](https://medium.com/javarevisited/10-advanced-open-source-command-line-tools-enhance-your-productivity-8e41829b7814), which it's a premium access article.

In this case, Is it okay keep this in dialy.dev as itsn't free or I should to report? If yes to report, which type of report? 🤔 

Thanks for quickly reply.

**Comment by sabdailydev** [comment]
@alancpazetto  Thanks for responding back with an example. In this case, this post shouldn't be on daily.dev since users can't read it for free. Hence you could report it as follows:
<img width="692" alt="Screenshot 2024-08-23 at 15 57 34" src="https://github.com/user-attachments/assets/123bbcc6-c678-4cf4-914f-e4d34a0b9d57">
Hope this clarifies!  Thanks 🙌
